class.jetpack-plan.php

```diff
@@ -23,6 +23,163 @@ class Jetpack_Plan {
 
 	const PLAN_OPTION = 'jetpack_active_plan';
 
+	const JETPACK_PLAN_MAP = array(
+		'personal-plan'  => array(
+			'jetpack_personal',
+			'jetpack_personal_monthly',
+		),
+		'premium-plan'   => array(
+			'jetpack_premium',
+			'jetpack_premium_monthly',
+		),
+		'business-plan'  => array(
+			'jetpack_business',
+			'jetpack_business_monthly',
+		),
+		'ecommerce-plan' => array(),
+	);
+
+	const WPCOM_PLAN_MAP = array(
+		'personal-plan'  => array(
+			'personal-bundle',
+			'personal-bundle-monthly',
+			'personal-bundle-2y',
+		),
+		'premium-plan'   => array(
+			'value_bundle',
+			'value_bundle-monthly',
+			'value_bundle-2y',
+		),
+		'business-plan'  => array(
+			'business-bundle',
+			'business-bundle-monthly',
+			'business-bundle-2y',
+		),
+		'ecommerce-plan' => array(
+			'ecommerce-bundle',
+			'ecommerce-bundle-monthly',
+			'ecommerce-bundle-2y',
+			'vip',
+		),
+	);
+
+	/**
+	 * Check if a site is a WordPress.com site.
+	 *
+	 * @access public
+	 * @static
+	 *
+	 * @return bool True if the site is a WordPress.com Simple Site.
```

Comment:
What about Atomic?

Reply:
Is `IS_WPCOM` defined for Atomic? I'll have to check that.

Reply:
It's not. `IS_WPCOM` is meant to check for simple sites. For all intents and purposes, Atomic sites are Jetpack sites.

If we do need to check for Atomic at any point, this can help:
https://github.com/Automattic/jetpack/blob/31efa189ad223c0eb7ad085ac0650a23facf9ef5/functions.global.php#L75